14-microbiome-diversity.Rmd

```diff
@@ -20,35 +21,66 @@ data("GlobalPatterns")
 se <- GlobalPatterns
 ```
 
-## Alpha diversity
+## Alpha diversity indices
+
+
+**_Alpha diversity_**, also sometimes interchangeably used with the
+term **_species diversity_**, summarizes the distribution of species
+abundances in a given sample into a single number that depends on
+species richness and evenness. Diversity indices measure the overall
+community heterogeneity. A number of ecological diversity measures are
+available. The Hill coefficient combines many standard indices into a
```

Comment:
Does the Hill coefficient require an introduction?

Reply:
Yes, I think it will be useful to have in the book since understanding the Hill index can be rather insightful about diversity indices in general.